static/src/js/components/CustomSelectWidget/CustomSelectWidget.jsx

```diff
@@ -46,6 +46,7 @@ const CustomSelectWidget = ({
   schema,
   selectOptions: propsSelectOptions,
   uiSchema,
+  options: oneOfEnums = { enumOptions: null },
```

Comment:
can this move to defaultProps?  That will clean up the code coverage for this line too, I think.